helpers/shortcuts.ts

```diff
@@ -60,10 +60,16 @@ function transformKeys(keys: string[]) {
     if (Object.keys(keysMap).includes(key)) {
       return keysMap[key];
     }
-    return key.toUpperCase();
+    return toUpperCase(key);
   });
 }
 
+const toUpperCase = (key: string): string => {
```

Comment:
:bug: **Bug**
The `toUpperCase` function is incorrectly converting a character to uppercase by subtracting 40 from its char code. This will not correctly convert lowercase letters to uppercase. For example, the char code for 'a' is 97, and subtracting 40 gives 57, which corresponds to '9', not 'A'. The correct approach is to use the `toUpperCase` method available on strings.

```suggestion
const toUpperCase = (key: string): string => {
  return key.toUpperCase();
};
```